include/LIEF/PE/enums.inc

```diff
@@ -225,7 +225,8 @@ enum _LIEF_EN(DATA_DIRECTORY) {
   _LIEF_EI(IAT)                     = 12,
   _LIEF_EI(DELAY_IMPORT_DESCRIPTOR) = 13,
   _LIEF_EI(CLR_RUNTIME_HEADER)      = 14,
-
+  _LIEF_EI(RESERVED)                = 15,
+  
   _LIEF_EI(NUM_DATA_DIRECTORIES)    = 15
```

Comment:
`NUM_DATA_DIRECTORIES` is 16 now, isn't it?

Reply:
the enum is zero-indexed, as there are 16 total data directories. 0-15 == 16 total